### DISCUSSION TITLE
Copyright Assignment

### DISCUSSION THREAD

**Discussion by jcsmith2** [nathanhoad/godot_dialogue_manager#397] [Q&A] [answered]
More than 30 GitHub accounts are associated with commits to this repository, but the copyright notice only refers to the primary author. Is there any evidence that all contributors willingly assigned their copyright to @nathanhoad so that it is safe to rely on the text of [LICENSE](https://github.com/nathanhoad/godot_dialogue_manager/blob/main/LICENSE)?

**Comment by nathanhoad** [comment, marked as answer]
I've updated the license to include the contributors 👍
